client/actions/projects.js

```diff
@@ -321,7 +323,10 @@ const syncProject = (dispatch, getState) => {
     })
     .then(response => {
       const patched = jsondiff.patch(state.savedProject, patch);
-      if (response.checksum !== shasum(omit(patched, ...response.checksumOmit))) {
+      // always exclude the id from the checksum since it does not exist in the server data and is set locally
+      // also exclude any server-computed properties that are returned in the response
+      const checksumOmit = [...(response.checksumOmit || []), 'id'];
+      if (response.checksum !== shasum(omit(patched, ...checksumOmit))) {
```

Comment:
`omit()` can take an array as the 2nd param so no need for the expansion here really.